CHANGELOG

```diff
@@ -1,4 +1,16 @@
 # Changelog
+## [0.2.8] - 2020-03-10
+### Bug Fixes
+ - Fix adding slave to newly create bridge causes bridge link down.
+ - Fix cannot switch bond mode from 1(active-backup) to 5(balance-tlb)
+
+### Known Issues with NetworkManager Version 1.22.8
+ - All changes to bridges might disrupt their connectivity.
```

Comment:
At least one bridge fix is included, are there more unfixed issues?

Reply:
I think yes, AFAIK, the fix included is only for slaves.